Sources/MongoSwift/BSON/CodableNumber.swift

```diff
@@ -34,10 +34,10 @@ extension CodableNumber {
         return nil
     }
 
-    /// By default, just try casting the number to a `BsonValue`. Types
-    /// where that will not work provide their own `asBsonValue` impl. 
-    var bsonValue: BsonValue? {
-        return self as? BsonValue
+    /// By default, just try casting the number to a `BSONValue`. Types
+    /// where that will not work provide their own `asBSONValue` impl. 
```

Comment:
@kmahar: I realize this last sentence predates @Utagai's change, but is "`asBSONValue` impl" technically correct? I read this assuming it was a protocol, but it appears to be referring to a type being extended with a `var bsonValue: BsonValue?` computed property. Would it make more sense to refer to "their own `bsonValue` computed property"?

Reply:
yes, I think I initially had named it `asBsonValue` when I wrote the comment, and later changed the name. suggesting fix to clarify.
```suggestion
    /// where that will not work provide their own implementation of the `bsonValue` computed property. 
```
@Utagai, you might want to fix the line break position in this comment, since this suggestion is a lot longer than the line before. but I can't figure out how to suggest for both lines at once.

Reply:
Done.